ports/psoc6/mpconfigport.h

```diff
@@ -220,6 +220,7 @@ extern void machine_deinit();
 
 #define MICROPY_LOGGER_DEBUG                    (0)
 
+
```

Comment:
Can this file be reverted as there are no changes here?

Reply:
done